apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -87,6 +87,35 @@ export class ApplicationService {
     }
   }
 
+  async findByNationalId(
+    nationalId: string,
+    municipalityCode: string,
+  ): Promise<ApplicationModel[]> {
+    return this.applicationModel.findAll({
+      where: {
+        [Op.or]: [
+          {
+            nationalId,
+            municipalityCode,
+          },
+          {
+            spouseNationalId: nationalId,
+            municipalityCode,
```

Comment:
Are we sure we should return the spouse? Maybe more a po question then a code one, I not quite sure that is why I am asking 😊